### PR TITLE
logger

### PR DESCRIPTION
**Pull request by bvdeenen**
For `logger` command, cheat sheet for the log levels and syntax

### CODE REVIEW

**Comment by chubin** [comment]
@bvdeenen You've probably ment `logger` in the last line?

File: sheets/logger [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+#              lpr
+#              mail
+#              news
+#              syslog
+#              user
+#              uucp
+#              local0
+#                to
+#              local7
+log -p <facility>.<level> -f <file>
```
Comment (chubin) [inline sheets/logger:31]: `logger` not `log`, correct?

Comment (bvdeenen) [reply sheets/logger:31]: correct,
sorry I got distracted by a phonecall

**Comment by chubin** [comment]
@bvdeenen I've cleaned it up a little bit
Feel free to add new examples of `logger` usage, as well as other commands and cheat sheets

**Pull request merged**